sources/packages/api/src/route-controllers/institution-locations/institution-location.aest.controller.ts

```diff
@@ -18,7 +17,7 @@ import {
  */
 @AllowAuthorizedParty(AuthorizedParties.aest)
 @Groups(UserGroups.AESTUser)
-@Controller("institution/location")
+@Controller("location")
```

Comment:
just curious why `institution` is removed?

Reply:
To have the routes, something as below when we append the Client tags

api/aest/institution/:institutionId - One institution
api/aest/institution/:institutionId/location - All locations
api/aest/location/:locationId - One location

api/institutions/institution/:institutionId - One institution
api/institutions/institution/:institutionId/location - All locations
api/institutions/location/:locationId - One location